package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vite-plugin-fable",
-  "version": "0.1.0",
+  "version": "0.1.1",
```

Comment:
This change should have been part of https://github.com/fable-compiler/vite-plugin-fable/pull/46